tools/serverpod_cli/lib/src/create/create.dart

```diff
@@ -907,3 +915,13 @@ void _copyModuleTemplates(
   );
   copier.copyFiles();
 }
+
+Script? _locateFlutterBuildScript(Directory serverDir) {
+  try {
+    final pubspecFile = File(p.join(serverDir.path, 'pubspec.yaml'));
+    final scripts = Scripts.fromPubspecFile(pubspecFile);
+    return scripts['flutter_build'];
+  } catch (e) {
+    return null;
+  }
+}
```

Comment:
Why not just allow the `PathNotFoundException` or malformed pubspec exceptions to be raised here? I feels very blunt to just squash all and return `null` 🤔